### ISSUE TITLE
Why don't next button always enabled when we dump mysql and the show the error to us?

### ISSUE THREAD

**Issue by enkracken** [dbeaver/dbeaver#16993]
#### System information: 
- Manjaro Linux 5.10 LTS
- DBeaver version 22.0.4.202205022238
- Just mariadb jdbc driver

#### Connection specification:
- MariaDB 10.8.3
- MySQL
- Do you use tunnels or proxies (SSH, SOCKS, etc)? no

#### Describe the problem you're observing:
I have added local client to /usr/bin and find the MariaDB 10.8.3. I choose it, output folder to home/myusername/Documents/Database but the next button still disabled.
![image](https://user-images.githubusercontent.com/86159208/177666092-6794f4bd-a07c-46ba-a01d-0b9c860c9fe4.png)

#### Steps to reproduce, if exist:
Just dump your database and the next button in the second page always disabled.

#### Include any warning/errors/backtraces from the logs
How can i find the error log if the button disabled and nothing error warning happen?

**Comment by E1izabeth** [comment]
Hello, @singkongjalar. It's the last page of wizard, so there is no next page, that's why `Next` button is disabled. To start the job you need to push `Start` button between `Back` and `Cancel`.

**Comment by E1izabeth** [comment]
Maybe we need to improve UX. We could revise the order of these buttons or replace `Next` button with `Start` instead of disabling `Next` button.

**Comment by enkracken** [comment]
> Maybe we need to improve UX. We could revise the order of these buttons or replace `Next` button with `Start` instead of disabling `Next` button.

Yeah. Make it straight to always use between next or start would make it better. Sorry for late reply. I don't have any word in my head

**Comment by sparrowme** [comment]
I would recommend NOT changing the action under a button's location (even if you change the label)  (muscle memory), but consider leaving the location alone and simply make it disappear if being disabled appears confusing.

**Comment by E1izabeth** [comment]
@sparrowme thank you for suggestions
